submodules/LIB/hardware/modules/iob_utils/iob_utils.py

```diff
@@ -27,6 +27,6 @@ def _copy_srcs(cls):
                 # Delete sources for this purpose
                 os.remove(
                     os.path.join(
-                        cls.build_dir, cls.PURPOSE_DIRS[purpose], "iob_utils.vh"  
```

Comment:
not understanding why git is flagging this as a change. the change was done 2 commits ago.